content/docs/operating/security.md

```diff
@@ -177,6 +181,24 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
+
+In the future, server-side TLS support will be rolled out to the different
+projects under the Prometheus organization. Those projects include Prometheus,
+Alertmanager, Pushgateway and the official exporters. The projects will share
+the same TLS library, which will be based on the Golang vanilla TLS library. We
+keep [Golang's default TLS
+parameters](https://golang.org/pkg/crypto/tls/#Config), with one exception: we
+explicitly disable TLSv1.0 and TLSv1.1.
+
+There is no plans at the moment to extend those settings. If you need older TLS
```

Comment:
The first sentence is a bit wishy washy, I'd remove it.

Reply:
Rephrased that.